Sources/PassageFlex/PassagePasskeyAuthentication.swift

```diff
@@ -21,7 +21,8 @@ internal struct PassagePasskeyAuthentication {
         let registrationRequest = try PasskeyRegistrationRequest.from(startResponse)
         let authController = PasskeyAuthorizationController()
         let credential = try await authController.requestPasskeyRegistration(
-            registrationRequest: registrationRequest
+            registrationRequest: registrationRequest,
+            includeSecurityKeyOption: authenticatorAttachment == .crossPlatform
```

Comment:
Lets default to `.platform` here.
```suggestion
            includeSecurityKeyOption: authenticatorAttachment == .platform
```

Reply:
This is not a default value. I'm setting the value of `includeSecurityKeyOption` as true if those two are equal

Reply:
omg sorry, I misread 🤦‍♂️